gateway/src/integration-test/java/com/hurence/webapiservice/historian/HistorianImportJsonVerticleIT.java

```diff
@@ -0,0 +1,216 @@
+                                    assertEquals(new JsonArray("[[4.1,1477895724888],[6.5,1477916924845]]"), datapoints3);
+                                    JsonObject doc4 = docs.getJsonObject(0);
+                                    assertEquals("openSpaceSensors.Temperature444", doc4.getString(TARGET_RESPONSE_FIELD));
+                                    JsonArray datapoints4 = doc4.getJsonArray(DATAPOINTS_RESPONSE_FIELD);
+                                    assertEquals(new JsonArray("[[0.0,1477895724888],[9.1,1477916924845]]"), datapoints4);
+                                    testContext.completeNow();
+                                });
+                            })
+                            .subscribe();
+                })
+                .subscribe();
+    }
+
+    @Test
+    @Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
+    void checkAddedTimeseriesChunks(VertxTestContext testContext) {
+        long time1 = 1477895624866L;
+        long time2 = 1477916224866L;
+        JsonArray params = new JsonArray().add(new JsonObject().put("name", "openSpaceSensors.Temperature555").put("points", new JsonArray().add(new JsonArray().add(time1).add(2.0)).add(new JsonArray().add(time2).add(4.0))));
+        historian.rxAddTimeSeries(params)
+                .doOnError(testContext::failNow)
+                .doOnSuccess(rsp -> {
+                    testContext.verify(() -> {
+                        JsonObject response = new JsonObject().put("status", "OK").put("message", "injected 2 points of 1 metrics in 1 chunks");
+                        assertEquals(rsp, response);
+                    });
+                })
+                .subscribe();
+        JsonObject params1 = new JsonObject().put("names", new JsonArray().add("openSpaceSensors.Temperature555"));
+        historian.rxGetTimeSeriesChunk(params1)
+                .doOnError(testContext::failNow)
+                .doOnSuccess(rsp -> {
+                    testContext.verify(() -> {
+                        LOGGER.info("responces : {}", rsp);
+                        long totalHit = rsp.getLong(RESPONSE_TOTAL_FOUND);
+                        assertEquals(1, totalHit);
+                        JsonArray docs = rsp.getJsonArray(RESPONSE_CHUNKS);
+                        assertEquals(1, docs.size());
+                        JsonObject doc1 = docs.getJsonObject(0);
+                        assertTrue(doc1.containsKey(RESPONSE_METRIC_NAME_FIELD));
+                        assertEquals("openSpaceSensors.Temperature555",doc1.getString(RESPONSE_METRIC_NAME_FIELD));
+                        assertTrue(doc1.containsKey(RESPONSE_CHUNK_START_FIELD));
+                        assertTrue(doc1.containsKey(RESPONSE_CHUNK_END_FIELD));
+                        assertTrue(doc1.containsKey(RESPONSE_CHUNK_ID_FIELD));
+                        assertTrue(doc1.containsKey(RESPONSE_CHUNK_SIZE_FIELD));
+                        assertEquals(2,doc1.getLong(RESPONSE_CHUNK_SIZE_FIELD));
```

Comment:
Test the value of all meta information : You can look at the current schema here https://github.com/Hurence/historian/blob/master/historian-resources/src/main/sh/create-historian-collection.sh
there are currently two version of the schema. Test schema 1 for the moment.

Reply:
the ckunks that i got in solr does not have all these fields, it only have 9 fields (7 that are added in the adding chunk process, and two autogenerated) and i can't test the fields "id" and "_version_" it seems that they are not the same every time

Reply:
Yes it is you that must add those field when creating the chunk. I'll show you how we do it in loader module monday.

Reply:
ok thanks